File: mobile/Emotracker/Emotracker.Core/Model/WebMessage.cs
using System;

namespace Emotracker.Core
{
	[Serializable]
	public class WebMessage
	{
		public static readonly string OK_RESULT = "OK";
		public static readonly string ERROR_RESULT = "ERROR";

		public String State { get; set; }
		public String Message { get; set; }
		public Object Result { get; set; }
	}
}

File: mobile/android/Emotracker/app/src/main/java/com/innerman/emotracker/model/network/WebMessage.java
package com.innerman.emotracker.model.network;

/**
 * Created by petrpopov on 08.03.14.
 */
public class WebMessage<T> {

    protected MessageState state;
    protected String message;
    protected T result;

    public MessageState getState() {
        return state;
    }

    public void setState(MessageState state) {
        this.state = state;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public T getResult() {
        return result;
    }

    public void setResult(T result) {
        this.result = result;
    }
}
